### PR TITLE
Add features to release notes

### PR DESCRIPTION
**Pull request by eduardoj**
Also fix style issues.

### CODE REVIEW

File: ReleaseNotes-2.9 [hunk 1]
```diff
@@ -63,6 +69,14 @@ Bugfixes:
  * Deletion of groups with users was not working
  * Fixed generation of the breadcrumb for the Image Editor (Kiwi)
  * Using ActiveSupport::JSON as encoder everywhere prevent to fail on Notification creation with events that have a very big payload
+ * Changes have been reduced in the Kiwi XML layout when modifications are made via the Kiwi Editor
+ * Request does not include element for change
+ * Kiwi importer drops directories
+ * Fix bsc#1068874 in OBS 2.8
```
Comment (adrianschroeter) [inline ReleaseNotes-2.9]: this is for the 2.9 release? ;)

Please don't mention single bugs, except they had been really a big issue.

And please don't put just the number here, but some description instead.

File: ReleaseNotes-2.9 [hunk 2]
```diff
@@ -63,6 +69,14 @@ Bugfixes:
  * Deletion of groups with users was not working
  * Fixed generation of the breadcrumb for the Image Editor (Kiwi)
  * Using ActiveSupport::JSON as encoder everywhere prevent to fail on Notification creation with events that have a very big payload
+ * Changes have been reduced in the Kiwi XML layout when modifications are made via the Kiwi Editor
```
Comment (adrianschroeter) [inline ReleaseNotes-2.9]: The editor did no exist before, so Editor is the feature, but we don't need to explain changes during the 2.9 development phase.

File: ReleaseNotes-2.9 [hunk 3]
```diff
@@ -96,7 +110,10 @@ Wanted changes:
 Other changes
 =============
 
- *
+ * Rubocop agreement, cleanup rubocop configuration and apply missing cops.
+ * Refactor of Event module.
+ * Move ProjectLogRotate to 1 event per job.
+ * Frontend test suite migration of several tests from minitests to rspec.
```
Comment (adrianschroeter) [inline ReleaseNotes-2.9]: This has no consequences for the admin, so it does not belong here.

Comment (eduardoj) [reply ReleaseNotes-2.9]: When you say "it" what are you referring to exactly? The last line or all the four lines?

When you say that "This has no consequences for the admin" do you mean that the section "Other changes" is used for... something related to "consequences for the admin"? Where do you suggest this four lines can be placed into?

File: ReleaseNotes-2.9 (hunk 1 above)
Comment (eduardoj) [reply ReleaseNotes-2.9]: Ops. Removed.

File: ReleaseNotes-2.9 (hunk 2 above)
Comment (eduardoj) [reply ReleaseNotes-2.9]: Ok. Lines not needed removed.

**Comment by adrianschroeter** [comment]
On Donnerstag, 1. Februar 2018, 16:26:38 CET wrote Eduardo J.:
> eduardoj commented on this pull request.
> 
> 
> 
> > @@ -96,7 +110,10 @@ Wanted changes:
>  Other changes
>  =============
>  
> - *
> + * Rubocop agreement, cleanup rubocop configuration and apply missing cops.
> + * Refactor of Event module.
> + * Move ProjectLogRotate to 1 event per job.
> + * Frontend test suite migration of several tests from minitests to rspec.
> 
> When you say "it" what are you referring to exactly? The last line or all the four lines?

I think it is true for all of them, yes.

> When you say that "This has no consequences for the admin" do you mean that the section "Other changes" is used for... something related to "consequences for the admin"? Where do you suggest this four lines can be placed into?

IMHO they don't belong to the release notes.
You may want to start a seperate developer changelog if you see the need.

The release notes should be limited to tell Admins and Users what kind of new functionality 
they will get. And point out to possible problems due to changes.

Also optimization and bugfixes is not something what belongs here.
This is what we cover usually with some generic sentence.
Except it was a big issue for former OBS instance users (and not
only a problem of us with our large server).

-- 

Adrian Schroeter
email: adrian@suse.de

SUSE Linux GmbH, GF: Felix Imendörffer, Jane Smithard, Graham Norton, HRB 21284 (AG Nürnberg) 
 
Maxfeldstraße 5                         
90409 Nürnberg 
Germany

**Comment by eduardoj** [comment]
@adrianschroeter , applied all your corrections. Please re-review the changes.

**Pull request merged**